recipes/hyperscan/all/patches/fix-cmake.patch

```diff
@@ -67,10 +67,11 @@ index 8bc6077..579a7b4 100644
  if (NOT CORRECT_PCRE_VERSION)
      message(STATUS "PCRE ${PCRE_REQUIRED_VERSION} or above not found")
  endif()
-@@ -483,16 +483,16 @@ if (CORRECT_PCRE_VERSION AND PCRE_BUILD_SOURCE AND BUILD_STATIC_LIBS)
+@@ -482,33 +482,18 @@ if (CORRECT_PCRE_VERSION AND PCRE_BUILD_SOURCE AND BUILD_STATIC_LIBS)
+     set(BUILD_CHIMERA TRUE)
  endif()
  
- add_subdirectory(unit)
+-add_subdirectory(unit)
 -if (EXISTS ${CMAKE_SOURCE_DIR}/tools/CMakeLists.txt)
 +if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/tools/CMakeLists.txt)
```

Comment:
Diff in diff view is hard to read --- but I think the old one is not require since the new toolchain does not add subdirectory

minor comment (this new code is better) just trying to cleanup